enhancements/installer/ovirt-rhv-upi.md

```diff
@@ -0,0 +1,166 @@
+---
+title: ovirt-rhv-upi
+authors:
+  - "@gekorob"
+  - "@rolfedh"
+reviewers:
+  - "@sdodson"
+  - "@abhinavdahiya"
+approvers:
+  - TBD
+creation-date: 2020-05-06
+last-updated: 2020-05-15
+status: implementable
+---
+
+# oVirt-RHV UPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Ovirt support for OpenShift 4 was initially focused on the installer provisioned
+(IPI) workflow.
+
+This enhancement proposes adding tooling and documentation to help users deploy
+OpenShift 4 on existing _user-provisioned infrastructure_ (UPI) in oVirt.
+
+## Motivation
+
+Most users who want to deploy OpenShift on oVirt/RHV have a highly-customized
+infrastructure. We can help these users simplify the process of installing OpenShift
+on oVirt using existing user-provisioned infrastructure such as DNS, DHCP, Load Balancers.
+
+Currently, users who want to deploy OpenShift on existing infrastructure in oVirt must
+adapt the [bare metal UPI instructions][baremetal-upi], which are not optimal for oVirt.
+
+Using the current installation program for deploying OpenShift 4 on oVirt does not work
+because it creates an opinionated set of _installer-provisioned infrastructure_ (IPI).
+Though convenient, this IPI does not meet the needs of users who want to use _existing_
+infrastructure.
+
+### Goals
+
+The main goal of the UPI is to grant users a higher degree of freedom compared to the
+more opinionated installer-provisioned installation.
+
+As a first step, we would create a provisioning installer codebase and a set of documented steps.
+These would enable users to deploy OpenShift on UPI in a way that is very similar to IPI but
+reusing and customizing pre-existing infrastructure elements.
+
+Following other UPI installer directives, this first step would include
+
+* Making oVirt-RHV UPI documentation available here:
+https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_upi.md
+* Making Ansible playbooks for scripts for oVirt/RHV resource creation available here:
+https://github.com/openshift/installer/tree/master/upi/ovirt
+* Making a CI job executing the provisioning scripts to test the UPI installer
+
+### Non-Goals
+
+It is outside the scope of this enhancement to provide explanations about the installation
+of infrastructure elements that are considered as required and owned by the user (e.g. DNS,
+DHCP, Load Balancer...)
+
+## Proposal
+
+* Write Ansible playbooks that can automate as much as possible the creation of ovirt resources
+like the VMs that will be used as masters and worker nodes of the cluster.
+* Write documentation and Ansible scripts to help users configure pre-existing infrastructure
+elements, such as DNS, DHCP
```

Comment:
How will we be configuring DNS, DHCP if we're not detailing how to install those components as described above in `Non-Goals` section?